crates/arkworks/src/group_affine.rs

```diff
@@ -15,17 +15,17 @@ use wasm_bindgen::prelude::*;
 #[wasm_bindgen]
 #[derive(Clone, Copy, Debug)]
 pub struct WasmGPallas {
-    pub x: WasmPastaFp,
-    pub y: WasmPastaFp,
-    pub infinity: bool,
+    pub(crate) x: WasmPastaFp,
```

Comment:
I'm wondering if it has an impact on the generated JavaScript code. Did you check?

Reply:
It seems to be used [here](https://github.com/o1-labs/o1js/blob/main/src/bindings/crypto/bindings/conversion-base.ts#L81).